### PR TITLE
Alternative dev workflow based on Docker.

### PR DESCRIPTION
**Pull request by crhym3**
Merge this as soon as we publish google/webfundamentals-dev Docker image.

R: @wibblymat

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -246,7 +251,7 @@ module.exports = function(grunt) {
       if (((release === undefined) && (err === undefined)) || (err)) {
         grunt.log.write('Failed to retrieve latest web-starter-kit release ');
         grunt.log.writeln('information - using wsk-version.json.');
-        
+
```
Comment (crhym3) [inline Gruntfile.js]: I didn't do it :)

**Comment by jpmedley** [comment]
Alex, do you have an ETA on that?

**Comment by crhym3** [comment]
On what, publishing the image? I'm ready to do it even now but it has to be approved internally.

**Comment by jpmedley** [comment]
New Readme Text: LGTM

**Comment by crhym3** [comment]
@jpmedley but you don't have to wait. You can build the image locally to try things out. There's a section in the readme on how to build the image.

**Pull request merged**